blivet/partitioning.py

```diff
@@ -1040,7 +1040,7 @@ def __init__(self, lv):
 
         # Round up to nearest pe. For growable requests this will mean that
         # first growth is to fill the remainder of any unused extent.
-        self.base = int(lv.data_vg_space_used // lv.vg.pe_size)
+        self.base = int(lv.size // lv.vg.pe_size)
```

Comment:
The fact that it makes sense. :) ``lv.size`` is a better value for ``base`` than ``data_vg_space_used`` which also includes the data part of the cache (if any) and potentially some other things (in future).